backend/tests/tests.js

```diff
@@ -0,0 +1,12 @@
+import config from '../config/config';
+
+if (!config.mock) {
+  throw '\n\n   Tests should only be run on the mock database. Please set the mock flag in the config to true.\n \
+  If you know what you are doing and wish to run this on a real database, remove this check.\n\n';
+}
+
+// List all tests to be run.
+require('./reservation/getAllForRestaurant'); // Using IDs 1-5
```

Comment:
Nitpick: Try not to use inline comments in the future (can be hard to read sometimes)

Reply:
I was thinking, as there will be multiple lines all importing a file and declaring an id, having the id comment above might get a bit confusing as to what test it belongs to.